sql/policy_internal.sql

```diff
@@ -127,14 +127,14 @@ BEGIN
     IF verbose_log THEN
        RAISE LOG 'job % completed processing chunk %.%', job_id, chunk_rec.schema_name, chunk_rec.table_name;
     END IF;
-    IF maxchunks > 0 AND numchunks >= maxchunks THEN
+    IF maxchunks > 0 AND numchunks_compressed >= maxchunks THEN
          EXIT;
     END IF;
   END LOOP;
 
   IF chunks_failure > 0 THEN
     RAISE EXCEPTION 'columnstore policy failure'
-      USING DETAIL = format('Failed to convert %L chunks to columnstore. Successfully converted %L chunks.', chunks_failure, numchunks - chunks_failure);
+      USING DETAIL = format('Failed to convert %L chunks to columnstore. Successfully converted %L chunks.', chunks_failure, numchunks_compressed);
```

Comment:
You might want to set an error code here as well since it will be "internal error" otherwise.

Reply:
Witch ERRCODE do u recommend? 

I think that at some point we should revisit src/errors.h and add more error states for our features, when necessary of course.

Reply:
No strong opinion, but some Data Exception error might make sense. This is what COPY uses (in some cases) when it cannot complete an operation, or just `ERRCODE_WARNING` (which is closer to what this actually is, even if it is reported as an error).

Reply:
ERRCODE_DATA_EXCEPTION makes sense.